ui/src/app/pages/access/data-access-requirements.tsx

```diff
@@ -363,11 +362,10 @@ const handleRasCallback = (
 
 const selfBypass = async (
   spinnerProps: WithSpinnerOverlayProps,
-  reloadProfile: Function,
-  modules: AccessModule[] = allInitialModules
```

Comment:
didn't need to update any callers because this is not used.  (I think it may have been used before we merged annual-renewal into here)